website/docs/tutorials/metatype-basics/index.mdx

```diff
@@ -381,12 +352,10 @@ typegates:
   dev:
     # ..
     secrets:
-      roadmap:  # <-- typegraph name
-        # values here assume default config
-        POSTGRES: "postgresql://postgres:password@postgres:5432/db"
+      TG_ROADMAP_POSTGRES: "postgresql://postgres:password@postgres:5432/db"
 ```
 
-Meta-cli will auto-reload when it detects changes to `metatype.yaml`.
+Meta-cli will auto-reload when it detects changes to `metatype.yaml` because Meta-cli was run in `dev` mode(through the `meta dev` command).
```

Comment:
nitpick: Rephrase it a bit? sentence feels awkward. Maybe break it in two.

Reply:
Done, broken to two.